hazelcast/src/test/java/com/hazelcast/cluster/ClusterShutdownTest.java

```diff
@@ -56,11 +57,19 @@ private void testClusterShutdownWithSingleMember(final ClusterState clusterState
         final Thread thread = new Thread(new Runnable() {
             @Override
             public void run() {
-                instances[0].getCluster().shutdown();
+                while (true) {
+                    try {
+                        instances[0].getCluster().shutdown();
+                        break;
+                    } catch (Exception ignored) {
```

Comment:
log why? this is a test and it's known that `shutdown()` can fail. Logging them will be just noise in the logs.